fvm/fvm_test.go

```diff
@@ -467,7 +467,7 @@ func TestEventLimits(t *testing.T) {
 
 	testContract := `
 	access(all) contract TestContract {
-		access(all) event LargeEvent(value: Int256, str: String, list: [UInt256], dic: {String: String})
+		access(all) event LargeEvent(value: Int256, str: Name, list: [UInt256], dic: {Name: Name})
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig